src/Android/Avalonia.Android/AvaloniaActivity.cs

```diff
@@ -1,35 +1,54 @@
 using Android.App;
+using Android.Content.Res;
 using Android.OS;
 using Android.Views;
+using AndroidX.AppCompat.App;
+using AndroidX.Lifecycle;
 
 namespace Avalonia.Android
 {
-    public abstract class AvaloniaActivity : Activity
+    public abstract class AvaloniaActivity : AppCompatActivity
```

Comment:
Is there any possible disadvantage of having AndroidX appcompat reference? 
I can assume users won't be able to use app compat activity features otherwise, if Avalonia won't have it there, right?

Reply:
None that I know of. most features added in newer sdks require AppCompat so as to backport them to older platforms. `Activity` is too restrictive as it only supports features for the api level the sdk is using, and the platform for that api level.

Reply:
Thanks